### PR TITLE
Match and label some track functions

### PR DESCRIPTION
**Pull request by FazanaJ**

### CODE REVIEW

File: src/unknown_0255E0.c [hunk 1]
```diff
@@ -266,94 +268,97 @@ void func_800249F0(u32 arg0, u32 arg1, s32 arg2, Vehicle vehicle, u32 arg4, u32
...
+        gCurrentLevelHeader2->unkAA = (gCurrentLevelHeader2->unkAA + (gCurrentLevelHeader2->unkA3 * tempUpdateRate)) & i;
+        func_8007EF80(gCurrentLevelHeader2->unkA4, &D_8011B114, &D_8011B110, tempUpdateRate);
     }
     flip = FALSE;
     if (get_filtered_cheats() & CHEAT_MIRRORED_TRACKS) {
         flip = TRUE;
     }
-    if (D_A0000200 != 0xAC290000) {
+    // Antipiracy measure
+    if (*((u32 *) 0xA0000200) != 0xAC290000) {
```
Comment (Ryan-Myers) [inline src/unknown_0255E0.c:331]:
```suggestion
    if (IO_READ(0x200) != 0xAC290000) {
```

File: src/unknown_0255E0.c [hunk 2]
```diff
@@ -367,26 +372,24 @@ void render_scene(Gfx** dList, MatrixS** mtx, Vertex** vtx, s8** tris, s32 updat
...
-        (get_current_level_race_type() != RACETYPE_CHALLENGE_EGGS) &&
-        (get_current_level_race_type() != RACETYPE_CHALLENGE_BATTLE) &&
-        (get_current_level_race_type() != RACETYPE_CHALLENGE_BANANAS)) {
-        if (get_multiplayer_hud_setting() == 0) {
+
+    if (numViewports == VIEWPORTS_COUNT_4_PLAYERS && get_current_level_race_type() != RACETYPE_CHALLENGE_EGGS && get_current_level_race_type() != RACETYPE_CHALLENGE_BATTLE && get_current_level_race_type() != RACETYPE_CHALLENGE_BANANAS) {
+        if (get_hud_setting() == 0) {
             if (flip) {
-                gSPSetGeometryMode(gSceneCurrDisplayList++, CVG_X_ALPHA);
+                gSPSetGeometryMode(gSceneCurrDisplayList++, 0x1000);
```
Comment (Ryan-Myers) [inline src/unknown_0255E0.c:388]: Should this be CVG_X_ALPHA or G_CULL_FRONT like above?

**Review by DavidSM64 — APPROVED** [review]
LGTM

**Review by Ryan-Myers — APPROVED** [review]

File: src/unknown_0255E0.c (hunk 2 above)
Comment (DavidSM64) [reply src/unknown_0255E0.c:388]: Ah good catch, that should be `G_CULL_FRONT`

Comment (FazanaJ) [reply src/unknown_0255E0.c]: yep already fixed

**Pull request merged**